test/gtest/uct/uct_test.cc

```diff
@@ -336,6 +335,19 @@ std::vector<const resource*> uct_test::enum_resources(const std::string& tl_name
 void uct_test::init() {
 }
 
+int uct_test::get_entity_index(entity *ent) {
+    int index = 0;
+
+    FOR_EACH_ENTITY(iter) {
```

Comment:
minor: entity is an element of `std::vector`, so can use smt like `std::distance(vec.begin(), std::find(vec.begin(), vec.end(), ent))`